internal/provider/resource_logspipeline.go

```
@@ -0,0 +1,232 @@
+
+// Create creates the resource and sets the initial Terraform state.
+func (r *logsPipelineResource) Create(ctx context.Context, req resource.CreateRequest, resp *resource.CreateResponse) {
+	// Read Terraform plan data into the model
+	var plan logsPipelineResourceModel
+	diags := req.Plan.Get(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	tflog.Debug(ctx, fmt.Sprintf("Creating LogsPipeline"))
```

golangci-lint (1.64.8)

S1039: unnecessary use of fmt.Sprintf

(gosimple)